python/ql/test/library-tests/ControlFlow/pruning/Constraint.expected

```diff
@@ -1,114 +1,130 @@
+| 38 | t5 | t5 | Truthy | test |
+| 39 | t6 | t6 | Truthy | test |
+| 40 | t6 | t6 | Truthy | test |
+| 43 | t1 | t1 | Truthy | test |
+| 44 | UnaryExpr | t2 | Falsey | test |
+| 44 | t2 | t2 | Truthy | test |
+| 47 | t1 | t1 | Truthy | test |
+| 48 | t2 | t2 | Truthy | test |
+| 49 | t2 | t2 | Truthy | test |
+| 51 | t2 | t2 | Truthy | test |
+| 52 | t2 | t2 | Truthy | test |
+| 55 | seq1 | seq1 | Truthy | test |
+| 57 | UnaryExpr | seq2 | Falsey | test |
+| 57 | seq2 | seq2 | Truthy | test |
+| 60 | seq1 | seq1 | Truthy | test |
+| 62 | seq1 | seq1 | Truthy | test |
+| 63 | seq2 | seq2 | Truthy | test |
+| 65 | seq2 | seq2 | Truthy | test |
+| 66 | seq3 | seq3 | Truthy | test |
+| 68 | UnaryExpr | seq4 | Falsey | test |
+| 68 | seq4 | seq4 | Truthy | test |
+| 71 | seq3 | seq3 | Truthy | test |
+| 73 | var | var | Truthy | test |
+| 74 | seq4 | seq4 | Truthy | test |
+| 76 | var | var | Truthy | test |
+| 78 | seq5 | seq5 | Truthy | test |
+| 80 | seq5 | seq5 | Truthy | test |
+| 81 | seq5 | seq5 | Truthy | test |
+| 83 | var | var | Truthy | test |
+| 88 | UnaryExpr | x | Falsey | test |
+| 88 | x | x | Truthy | test |
+| 89 | Exception | Exception | Truthy | test |
+| 90 | y | y | Truthy | test |
+| 91 | Exception | Exception | Truthy | test |
+| 92 | make_a_call | make_a_call | Truthy | test |
+| 93 | UnaryExpr | x | Falsey | test |
+| 93 | x | x | Truthy | test |
+| 94 | count | count | Truthy | test |
+| 95 | y | y | Truthy | test |
+| 96 | count | count | Truthy | test |
+| 99 | another_module | another_module | Truthy | assign |
+| 101 | make_a_call | make_a_call | Truthy | test |
+| 102 | UnaryExpr | another_module | Falsey | test |
+| 102 | another_module | another_module | Truthy | test |
+| 103 | count | count | Truthy | test |
+| 107 | UnaryExpr | t1 | Falsey | test |
+| 107 | t1 | t1 | Truthy | test |
+| 109 | t2 | t2 | Truthy | test |
+| 111 | t1 | t1 | Truthy | test |
+| 113 | UnaryExpr | t2 | Falsey | test |
+| 113 | t2 | t2 | Truthy | test |
+| 117 | UnaryExpr | test | Falsey | test |
+| 117 | test | test | Truthy | test |
+| 119 | UnaryExpr | test | Falsey | test |
+| 119 | test | test | Truthy | test |
+| 123 | m | m | Truthy | test |
+| 125 | m | m | Truthy | test |
+| 126 | m | m | Truthy | test |
+| 158 | Compare | ps | Is not None | test |
+| 158 | ps | ps | Truthy | test |
+| 159 | ps | ps | Truthy | test |
+| 160 | Compare | ps | Is None | test |
+| 160 | ps | ps | Truthy | test |
+| 171 | __name__ | __name__ | Truthy | test |
+| 172 | None | None | Truthy | test |
+| 172 | escapes | escapes | Is None | assign |
+| 174 | func | func | Truthy | test |
+| 175 | Exception | Exception | Truthy | test |
+| 176 | count | count | Truthy | test |
+| 177 | Compare | escapes | Is None | test |
+| 177 | None | None | Truthy | test |
+| 177 | escapes | escapes | Truthy | test |
+| 178 | count | count | Truthy | test |
+| 180 | count | count | Truthy | test |
+| 188 | true12 | true12 | Truthy | test |
```

Comment:
This one surprises me somewhat. How is `true12` being tested on the line in question? It gets called (and inside a `try` block) but apart from that, I can't see how it would affect control flow. (In particular, I don't see why it would necessarily be `Truthy`.)

Reply:
The `constraintFromTest` wasn't just for tests, but all expressions.Fixed.